Britannia/common/decisions/BTWK_religious_decisions.txt

```diff
@@ -0,0 +1,241 @@
+					add_character_modifier = BTWK_frankish_deity_tyr_modifier
+				}
+				100 = {
+					trigger = {
+						has_faith = faith:frankish_pagan
+					}
+					show_chance = no
+					desc = select_personal_deity_germanic_decision.select_thor
+					add_character_modifier = BTWK_frankish_deity_thor_modifier
+				}
+
+				# SUEBIC
+				100 = {
+					trigger = {
+						has_faith = faith:suebic_pagan
+					}
+					show_chance = no
+					desc = select_personal_deity_germanic_decision.select_odin
+					add_character_modifier = BTWK_suebic_deity_odin_modifier
+				}
+				100 = {
+					trigger = { 
+						has_faith = faith:suebic_pagan
+						is_old_saxon_culture_trigger = no
+					}
+					show_chance = no
+					desc = select_personal_deity_germanic_decision.select_tyr
+					add_character_modifier = BTWK_suebic_deity_tyr_modifier
+				}
+				100 = {
+					trigger = {
+						has_faith = faith:suebic_pagan
+					}
+					show_chance = no
+					desc = select_personal_deity_germanic_decision.select_thor
+					add_character_modifier = BTWK_suebic_deity_thor_modifier
+				}
+
+				# GOTHIC
+				100 = {
+					trigger = {
+						has_faith = faith:gothic_pagan
+					}
+					show_chance = no
+					desc = select_personal_deity_germanic_decision.select_odin
+					add_character_modifier = BTWK_gothic_deity_odin_modifier
+				}
+				100 = {
+					trigger = { 
+						has_faith = faith:gothic_pagan
+						is_old_saxon_culture_trigger = no
+					}
+					show_chance = no
+					desc = select_personal_deity_germanic_decision.select_tyr
+					add_character_modifier = BTWK_gothic_deity_tyr_modifier
+				}
+				100 = {
+					trigger = {
+						has_faith = faith:gothic_pagan
+					}
+					show_chance = no
+					desc = select_personal_deity_germanic_decision.select_thor
+					add_character_modifier = BTWK_gothic_deity_thor_modifier
+				}
+			}
+		}
+		trigger_event = religious_decision.0102
+	}
+
+	ai_check_interval = 96
+	
+	ai_potential = {
+		piety >= minor_piety_value
+		NOR = {
+			has_character_modifier = BTWK_norse_deity_odin_modifier
+			has_character_modifier = BTWK_norse_deity_tyr_modifier
+			has_character_modifier = BTWK_norse_deity_thor_modifier
+
+			has_character_modifier = BTWK_anglo_saxon_deity_odin_modifier
+			has_character_modifier = BTWK_anglo_saxon_deity_tyr_modifier
+			has_character_modifier = BTWK_anglo_saxon_deity_thor_modifier
+			has_character_modifier = BTWK_deity_seaxneat_modifier
+
+			has_character_modifier = BTWK_frankish_deity_odin_modifier
+			has_character_modifier = BTWK_frankish_deity_tyr_modifier
+			has_character_modifier = BTWK_frankish_deity_thor_modifier
+
+			has_character_modifier = BTWK_suebic_deity_odin_modifier
+			has_character_modifier = BTWK_suebic_deity_tyr_modifier
+			has_character_modifier = BTWK_suebic_deity_thor_modifier
+
+			has_character_modifier = BTWK_gothic_deity_odin_modifier
+			has_character_modifier = BTWK_gothic_deity_tyr_modifier
+			has_character_modifier = BTWK_gothic_deity_thor_modifier
+
```

Comment:
Spacing?

Reply:
Whitespace at the bottom was just a mistake. Put spacing between the difference faiths' modifiers just to make it easier to read.